### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by alishaikh9890**
.

### CODE REVIEW

File: src/components/AddProgram/AddProgram.styles.ts [hunk 1]
```diff
@@ -0,0 +1,126 @@
+const addProgramStyles = {
+
```
Comment (amirmursal) [inline src/components/AddProgram/AddProgram.styles.ts:2]: remove extra line use prettier extension for VS code that we format your code on save

File: src/components/AddProgram/AddProgram.styles.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                fontSize:"30px"
+            },
+
+            "& tr > td:first-child":{
+                backgroundColor:"white",
+                color:"rgb(113, 173, 71)",
+            }
+
+
+
```
Comment (amirmursal) [inline src/components/AddProgram/AddProgram.styles.ts:121]: remove all the extra lines

File: src/components/AddProgram/DatePicker.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+import { UseDateFieldProps } from '@mui/x-date-pickers/DateField';
+import {
+  BaseSingleInputFieldProps,
+  DateValidationError,
+  FieldSection,
+} from '@mui/x-date-pickers/models';
+
+import DatePickerStyles from "./DatePicker.styles";
+
+interface ButtonFieldProps
```
Comment (amirmursal) [inline src/components/AddProgram/DatePicker.tsx:16]: create separate DatePicker.type.ts and add this interface in it

Comment (alishaikh9890) [reply src/components/AddProgram/DatePicker.tsx:16]: not able to resolve

**Review by amirmursal — APPROVED** [review]

**Pull request merged**